src/main/java/org/snomed/snowstorm/rest/ConceptController.java

```diff
@@ -580,6 +589,135 @@ public ExpressionStringPojo getConceptNormalForm(
 
 		return new ExpressionStringPojo(expression.toString(includeTerms));
 	}
+	
+	@PostMapping(value = "/{branch}/concepts/copy")
+	@PreAuthorize("hasPermission('AUTHOR', #branch)")
+	@JsonView(value = View.Component.class)
+	public Collection<ConceptMini> copyConcepts(
+			@ApiParam(value = "Destination branch where the concepts are copied to.")
+			@PathVariable String branch,
+			@ApiParam(value = "Source branch where the concepts are selected from.")
+			@RequestParam String sourceBranch,
+			@ApiParam(value = "ECL expression for selecting concepts to copy between the branches.")
+			@RequestParam String ecl,
+			@ApiParam(value = "Include dependant components. Defaults to false.")
+			@RequestParam(required = false, defaultValue = "false") boolean includeDependencies,
```

Comment:
Hi @astro-snail , it looks like you have not implemented this flag. Is it intentional?

Reply:
Hi @QuyenLy87 yes, as it is not required to be implemented initially, as far as I understand. Will need more clarification on what are the dependencies in this case.

Reply:
@QuyenLy87 I did implement the `includeDependencies` flag now.